pipeline-runner/R/qc-7-embed_and_cluster.R

```diff
@@ -4,6 +4,16 @@
 # done also in this step.
 
 embed_and_cluster <- function(scdata, config, sample_id, task_name = 'configureEmbedding') {
+  message("starting clusters")
+  type <- config$clusteringSettings$method
+  methodSettings <- config$clusteringSettings$methodSettings[[type]]
+  message("Running clustering")
+  res <- runClusters(type,methodSettings$resolution,scdata)
```

Comment:
naming suggestion: change `res` to `resolution`, because in other parts of the code very often you use `res` and mean result